### DISCUSSION TITLE
Welcome to Prowler Discussions!

### DISCUSSION THREAD

**Discussion by toniblyx** [prowler-cloud/prowler#981] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JArmandoG** [comment]
Hi! Is the Wazuh integration still on the roadmap to be taken again? I'm trying to fix the /var/ossec/integrations/prowler-wrapper.py but I'm having some issues

**Comment by JArmandoG** [comment]
Hi! I loved the Shodan lookup for Public IP/EIP! The "remediation" says, "Check Identified IPs; consider changing them to private ones and **_delete them from Shodan._**" - Why does it say delete them from Shodan? One does not simply go and "delete" an IP from Shodan.
